Xamarin.Forms.Platform.Android/CollectionView/ItemContentView.cs

```diff
@@ -6,7 +6,7 @@ namespace Xamarin.Forms.Platform.Android
 {
 	public class ItemContentView : ViewGroup
 	{
-		protected IVisualElementRenderer Content;
+		internal protected IVisualElementRenderer Content;
```

Comment:
Does this need to be marked `internal protected`? That might be a breaking change for folks who have subclassed `ItemContentView`.

Reply:
yeah i need to check the Element, maybe add this way:

```
	internal Element Element => Content`?.Element;
``